pkg/ingester/limiter.go

```diff
@@ -198,11 +211,34 @@ func (l *Limiter) convertGlobalToLocalLimit(globalLimit int) int {
 
 	// May happen because the number of ingesters is asynchronously updated.
 	// If happens, we just temporarily ignore the global limit.
-	if numIngesters > 0 {
-		return int((float64(globalLimit) / float64(numIngesters)) * float64(l.replicationFactor))
+	if numIngesters == 0 {
+		return 0
```

Comment:
So now local limit is 0 if num healthy ingesters is zero, right? I think that's probably okay, but the comment above is confusing now, since that's not what "ignore the global limit" makes me think.

Reply:
0 means unlimited.

Reply:
I see, thanks.

Reply:
Let me also outline the logic hasn't changed. If the ring client doesn't see any ingester (0), the global rate limit was temporarily disabled even before.